src/components/EditAbilityScores.js

```diff
@@ -0,0 +1,135 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+import 'airbnb-js-shims' // to allow jest to understand Object.entries for parsing the character objects
+
+import { Form, FormGroup, Col, Button, ButtonGroup, ControlLabel } from 'react-bootstrap'
+
+export default class EditAbilityScores extends Component {
+  constructor (props) {
+    super(props)
+
+    this.renderAbilityScorePackages = this.renderAbilityScorePackages.bind(this)
+    this.pointsRemaining = this.pointsRemaining.bind(this)
+    this.increaseScore = this.increaseScore.bind(this)
+    this.decreaseScore = this.decreaseScore.bind(this)
+    this.resetScores = this.resetScores.bind(this)
+  }
+
+  pointsRemaining () {
+    let points = 27
```

Comment:
Magic numbers like `27` are mysterious when showing up in code at random like this. A person might scratch their head and wonder where it came from and if it can be changed. A good practice is to define the magic number at the top of the module, e.g.:

```
const STARTING_POINTS = 27
...

  let points = STARTING_POINTS
```